v1.0/conformance_test_v1.0.yaml

```diff
@@ -152,14 +152,14 @@
     output: 2
   doc: Testing the string 'null' does not trip up an Any without a default value.
 
-- job: v1.0/wc-job.json
+- job: v1.0/cat-job.json
```

Comment:
What would be the impact of removing the `wc-*` files?

Reply:
They are necessary for a bunch of tests, so I decided not to remove them. Those tests seem to be stable across platforms.